pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+  // useState returns a value, and a function to set that value
+  const [sex, setSex] = useState(null as Number); // Default: null
+  const [albendazole, setAlbendazole] = useState(0); // Default: No
+  const [visit, setVisit] = useState(0); // Default: No
+  const [pregnant, setPregnant] = useState(0); // Default: No
+  const [planning, setPlanning] = useState(0); // Default: No
+
+  // Data for Radio Buttons
+  const sexValues = [
+    { value: 0, label: "Male" },
+    { value: 1, label: "Female" },
+  ];
+
+  const radioYesNo = [
+    { value: 0, label: "No" },
+    { value: 1, label: "Yes" },
+  ];
+
+  /* istanbul ignore next */
+  const onSubmit = () => {
+    // This shows how to access the values of the form
+    const firstNameField: HTMLInputElement = document.getElementById(
+      "firstname"
+    ) as HTMLInputElement;
+    const lastNameField: HTMLInputElement = document.getElementById(
+      "lastname"
+    ) as HTMLInputElement;
+    const nicknameField: HTMLInputElement = document.getElementById(
+      "nickname"
+    ) as HTMLInputElement;
+    const birthdateField: HTMLInputElement = document.getElementById(
+      "birthdate"
+    ) as HTMLInputElement;
+    const complaintField1: HTMLInputElement = document.getElementById(
+      "complaint1"
+    ) as HTMLInputElement;
+    const complaintField2: HTMLInputElement = document.getElementById(
+      "complaint2"
+    ) as HTMLInputElement;
+    const complaintField3: HTMLInputElement = document.getElementById(
+      "complaint3"
+    ) as HTMLInputElement;
+
+    console.log("First name:", firstNameField.value);
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
+
+  return (
+    <div id="patient-info-wrapper">
+      <TextareaItem
+        title="First Name"
+        placeholder="First Name"
+        autoHeight
+        id="firstname"
+      />
+
+      <TextareaItem
+        title="Last Name"
+        placeholder="Last Name"
+        id="lastname"
+        autoHeight
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
```

Comment:
Add constrain for only numerals, can currently enter alpha characters

Reply:
@abpwrs @bmitchinson If someone has some general suggestions for this field, that would be appreciated. I had originally intended on using the [Ant Design DatePicker](https://ant.design/components/date-picker/) but per feedback, I decided that was too much work with a mouse to scroll to the correct date. Ideally I would like to have something like [this](https://material-ui.com/components/pickers/#datepickers) but I was unable to find a way to implement that within the Ant Design components we are already using.

Finding a solution for adding the slashes to the date field would also help us resolve the Blood Pressure field.

Reply:
Resolved in [this commit](Birthdate allows digits + '-' only).

Reply:
@meghanmc The commit I added above allows for digits and `-` characters. Further validation on that object will have to happen with a regex, which could be applied with the "pattern" html parameter on the input tag contained in InputItem.

Reply:
The pattern attribute only blocks form submission if a regex don't match. It doesn't block input. To block input, we need to follow an example like so: https://mobile.ant.design/components/input-item/#components-input-item-demo-error

Reply:
Also added a constraint for 10 digits per the format we're currently using in [this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/668419e8e551d7956be1ed58edf8826dc397dd3f). I also added this constraint to LMP field.

Reply:
Looks good. There are still some edge cases that would require a lookahead regex, but this input validation is a good first iteration!